tests/release_version_manager.py

```diff
@@ -0,0 +1,5 @@
+import requests
+
+def latest_react_native_github_release():
+    react_native_releases = requests.get("https://api.github.com/repos/sentry-demos/sentry_react_native/releases")
+    return react_native_releases.json()[0]['tag_name']
```

Comment:
So this gets all of the releases and the 0th one is the most recent? Confirming. @cstavitsky

Reply:
ah duh the name of the method is latest...release, so that works! No need to answer me here.